indicator_templates/quarto/1_open_science/citizen_science.qmd

```diff
@@ -54,11 +56,18 @@ Notably principle 7 also claims that "citizen science project data and metadata
 -   the conservation and sharing of the resources generated by the project \[8. ownership\]
 -   the recognition as authors/protagonists of the research \[9. credit\]
 
-This document describes a series of metrics to quantify (but also qualify) each of these nine different forms of citizen participation in science, as well as a tenth indicator accounting for the capacity of a citizen sciences project to span across multiple forms of participation \[10. span\].
+This document describes a series of metrics to quantify (but also qualify) each of these nine different forms of citizen participation in science, as well as a tenth indicator accounting for the capacity of a citizen sciences project to span across multiple forms of participation \[10. Span\].
+
+## Open Science in European research programmes
+
+Citizen Science is possibly the most studied part of Open Science and one that has been heavily supported by the European Union in the FP7 and Horizon 2020 programs. At the time of writing (november 2024), a search for Citizen Science in the Cordis database (https://cordis.europa.eu) returns 93 programmes and 443 projects.
 
-## Existing datasources
+Beyond direct project funding, the EU also supports citizen science through organizations like the European Citizen Science Association (ECSA), which plays an important role as the hub for the scientific community using this approach [@vohland_citizen_2021]. 
+Founded in 2013 ECSA supports a network of citizen science initiatives, promoting high standards, shared methodologies, and collaboration opportunities. Its platform is centralize training material, resources, and guidelines for citizen scientists. 
```

Comment:
Unclear phrasing, see suggested fix below, but I'm not 100% sure if that is what you want to say.

```suggestion
Founded in 2013 ECSA supports a network of citizen science initiatives, promoting high standards, shared methodologies, and collaboration opportunities. Its platform centralises training material, resources, and guidelines for citizen scientists. 
```

Reply:
Agreed!